src/test/java/io/cdap/plugin/servicenow/sink/ServiceNowSinkOutputFormatProviderTest.java

```diff
@@ -0,0 +1,25 @@
+package io.cdap.plugin.servicenow.sink;
+
+import io.cdap.plugin.servicenow.sink.output.ServiceNowOutputFormatProvider;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.oltu.oauth2.common.exception.OAuthProblemException;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ServiceNowSinkOutputFormatProviderTest {
+
+  @Test
+  public void testGetOutputFormatClassName() throws OAuthProblemException {
+    Configuration configuration = new Configuration();
+    ServiceNowOutputFormatProvider serviceNowOutputFormatProvider = new ServiceNowOutputFormatProvider(configuration);
+    Assert.assertEquals("io.cdap.plugin.servicenow.sink.output.ServiceNowOutputFormat",
+                        serviceNowOutputFormatProvider.getOutputFormatClassName());
+  }
+
+  @Test
+  public void testGetOutputFormatConfiguration() throws OAuthProblemException {
+    Configuration configuration = new Configuration();
+    ServiceNowOutputFormatProvider serviceNowOutputFormatProvider = new ServiceNowOutputFormatProvider(configuration);
+    Assert.assertEquals(241, serviceNowOutputFormatProvider.getOutputFormatConfiguration().size());
```

Comment:
Instead of checking size() it would be better to check the contents of the map

Reply:
Done

Reply:
remove the size check, and check that the map contains the expected data.
The config that gets passed in is hConf - https://github.com/data-integrations/servicenow-plugins/blob/develop/src/main/java/io/cdap/plugin/servicenow/sink/ServiceNowSink.java#L84

Reply:
Done